src/scss/variables.scss

```diff
@@ -63,11 +63,11 @@ $font-size-large:         floor(($font-size-base * 1.125)) !default; // ~18px
 $font-size-small:         floor(($font-size-base * 0.875)) !default; // ~14px
 
 $font-size-h1:            floor(($font-size-base * 2.5)) !default; // ~40px
-$font-size-h2:            floor(($font-size-base * 2.125)) !default; // ~34px
-$font-size-h3:            floor(($font-size-base * 1.75)) !default; // ~28px
+$font-size-h2:            floor(($font-size-base * 2.0)) !default; // ~32px
```

Comment:
lint-sass hates the `.0` on the end [here](https://travis-ci.com/silverstripe/cwp-watea-theme/builds/87714151#L2896)
It looks nice I agree - I'll leave it up to you whether you choose to disable this option for this line (if that's possible), or just change it to read `floor(($font-size-base * 2))`

Reply:
Fixed